docker/entrypoint.sh

```diff
@@ -89,8 +89,16 @@ if [ "${RUN_CODEMETA}" == "true" ]; then
 fi
 
 echo $COMMAND
+
+# First time might just create content
 $COMMAND
 
+# Run twice to get additional metadata
+if [ "${INPUT_RUN_TWICE}" == "true" ]; then
```

Comment:
that is the one to change the 'default' right?

```suggestion
if [ "${INPUT_RUN_TWICE:-true}" == "true" ]; then
```

Reply:
`INPUT_RUN_TWICE` will always be set by the default in the action.yml  corresponding to `run_twice`. The convention when you have a Dockerfile for an action is names in action.yml are:

- converted all caps
- prefixed with `INPUT_`
- dash change to underscore

so `run_twice` == `INPUT_RUN_TWICE`, so that would need to change as well - but this chage is also good in case a user defines the variable and unsets it (feel free to commit that suggestion).